developers/integration.rst

```diff
@@ -125,9 +139,9 @@ approximately 5,000 masternode servers. These nodes are differentiated
 from standard nodes by having proven ownership of 1,000 Dash, making the
 network `highly resistant to Sybil attacks <https://en.wikipedia.org/wiki/Sybil_attack>`__. 
 Masternodes form `Long-Living Masternode Quorums (LLMQs) <https://github.com/dashpay/dips/blob/master/dip-0006.md>`__, 
-which are responsible for providing near instant certainty to the transaction
+which are responsible for providing near-instant certainty to the transaction
 participants that the transaction inputs cannot be respent, and that the
-transaction will be included in a following block instead of a conflicting
+transaction will be included in the following block instead of a conflicting
```

Comment:
The original was correct. A bit of IS nuance that may not be obvious. It doesn't guarantee the tx will be in a particular block, but that the inputs it spends can't be included in a different tx.